packages/kit/src/exports/public.d.ts

```diff
@@ -1159,6 +1178,8 @@ export interface SSRManifest {
 		nodes: SSRNodeLoader[];
 		routes: SSRRoute[];
 		matchers(): Promise<Record<string, ParamMatcher>>;
+		/** A `[file]: size` map of all assets imported by server code */
+		server_assets: Record<string, number>;
```

Comment:
if someone used a glob import, would this potentially turn into a very large list? do we actually need all filenames if only few might ever be passed through `read`?

Reply:
Yes but this is just a caveat of using `import.meta.glob` _anyway_ — you will always end up with filenames written into the resulting bundle, so you'd better make sure you're only globbing things that you will actually use